gateway/src/integration-test/resources/http/grafana/annotation/test1/expectedResponse.json

```diff
@@ -0,0 +1,26 @@
+{
+  "annotations" : [
```

Comment:
I would like more specific file name for the tests than 'test1', 'test2'... Choose a name depending on what you want to test. For example le limit of tags. 'LimitNumberOfTags', 'matchAnyTags', 'allTags', 'mathchingAllTag', 'mathchingAllTag2', 'dateRangeFilter'.
You can use number but specify a minimum what you test on the file so this is easily to understand for others programmers.